config/grumphp.yml

```diff
@@ -8,6 +8,7 @@ parameters:
     check_file_permissions: ~
     php_check_syntax: ~
     phpcs: ~
+    phpstan: ~
```

Comment:
If using Custom made task - task name is composed from Class name where PascalCase is transformed to snake_case so in this instance the built in `phpstan ` is used and not custom `php_stan`. This leads to some configuration to be broken. Either Class names should be changed or configurations and documentation needs to be updated.

Reply:
Thank you for the clarification!

Task name renamed from PhpStan to PhpstanCheckDeprecation to better describe what task is for.